README.md

```diff
@@ -0,0 +1,35 @@
+# CASS-Degrees-Code
+Primary Codebase for the TechLauncher CASS Degrees project
+
+For our landing page please go to: https://github.com/cass-degrees/CASS-Degrees
+
+## Our Git Branching Structure
+In this project we will have two main branches: **Master** and **Develop**
+
+The **Master** branch will only contain completed and working releases that have addressed all the user stories of the relevant release. 
+Merges to this branch will only come from the develop branch once a release in develop has been completed and finalised.
+
+The **Develop** branch is the main working branch which will have completed and working user stories for the relevant release that the team is working on.
+No one is to directly commit to this branch, it is updated by merges from branches created for the relevant user story or feature that a developer is working on.
+
+The idea is that once someone completes a feature or user story, they will create a new branch with their code (with an appropriate name) and request a *Pull Request* into
+the develop branch.
+
+**Pull Requests** are reviewed by the projects team members, the rule is that 2 people must approve of the feature or user story before it can
+be merged into the develop branch. Alternatively, if it has been seen by the project team members and has been open for more than 2 days, then the
+developer can merge into the develop branch, given that they have tested and ensured that their code is functional. 
+
+
+Please see this document for a full overview and detailed explanation of our branching structure: https://docs.google.com/document/d/1whU7j2F_0RFJ1n1rJsn3H6UcKvLCPCmKZ2J65a7D-9g/edit
+
+## Project Environment Setup 
+Please view this link: https://docs.google.com/document/d/1AP6ycdfm0Rk04vRhcsrh3G53PR-BPytMMqTyLkMv7vQ/edit?usp=sharing
```

Comment:
This should be moved into this repo entirely IMO - build instructions might differ between versions, and it should be versioned directly.

Reply:
Fair enough but I thought it should be here for now as it is our current project environment setup guide and it will help people who want to play with our program.
In the next release, we can update this link/information if anything has changed.

Reply:
I just realised I misinterpreted what you said. 
It would be good to have the instructions in the readme but I feel like the current instructions are too long and having it isolated is good for if we change things as we don't need to update the readme and the setup document.

Reply:
If the instructions are too long for the readme, chuck them in a separate markdown file. They are intrinsically attached to the codebase at particularly points.